data/portals.json

```diff
@@ -0,0 +1,13 @@
+{
+  "url": "https://github.com/WICG/portals/blob/master/explainer.md",
+  "title": "Portals",
+  "wgs": [
```

Comment:
This can be removed now, since #343 has been merged.

Reply:
Actually, it cannot for the time being because the code only associates `wicg.github.io` URLs to WICG and this one is `github.com/WICG`. I created #348 to extend the logic.

Reply:
Patch in #349, I'll drop `wgs` from this pull request (and pending others) accordingly.